### PR TITLE
helpers: Add --slient option to bash-it update

### LINKED ISSUE #1523: How do I set bash-it to update automatically and unattended?
Ideally I'd like to be able to create a `cron` job that runs `bash-it update` daily. However, I can't find any way to do run that command silently/non-interactively.

Any ideas? Is that even possible?

### PR DESCRIPTION
**Pull request by NoahGorny**
Resolves #1523 

Lemme know what you think @nwinkler :smile:

### CODE REVIEW

**Comment by NoahGorny** [comment]
Ping @nwinkler

File: lib/helpers.bash [hunk 1]
```diff
@@ -77,7 +77,8 @@ bash-it ()
         _bash-it-search $component "$@"
         return;;
       update)
-        func=_bash-it_update;;
+        _bash-it_update $component "$@"
```
Comment (nwinkler) [inline lib/helpers.bash:80]: Why did you add the `$component` here? Is this really needed for the update scenario?

File: lib/helpers.bash [hunk 2]
```diff
@@ -149,10 +150,31 @@ _bash-it-plugins ()
...
+  fi
+}
+
 _bash-it_update() {
   _about 'updates Bash-it'
   _group 'lib'
 
+  declare silent
+  for word in $@; do
+    if [[ ${word} == "--silent" || ${word} == "-s" ]]; then
```
Comment (nwinkler) [inline lib/helpers.bash:174]: Please add documentation for the two new options to the `README.md` file in the `Updating` section.

Comment (nwinkler) [inline lib/helpers.bash:174]: Should we also add the new `--silent` option to the `bash-it.completion.bash` file? The `update` command is not covered at the moment, since it did not have any options so far.

File: lib/helpers.bash (hunk 1 above)
Comment (NoahGorny) [reply lib/helpers.bash]: removed in the currrent version

File: lib/helpers.bash (hunk 2 above)
Comment (NoahGorny) [reply lib/helpers.bash:188]: done

Comment (NoahGorny) [reply lib/helpers.bash:188]: done

**Comment by NoahGorny** [comment]
@nwinkler, I moved this PR to [here](https://github.com/NoahGorny/bash-it/pull/1)
You can start by reviewing the moved PR or #1626
